android/app/src/main/java/com/microsoft/reactnativecodepush/CodePushInstallMode.java

```diff
@@ -0,0 +1,15 @@
+package com.microsoft.reactnativecodepush;
+
+public enum CodePushInstallMode {
+    IMMEDIATE(0),
```

Comment:
Do we need to explicitly set the numeric values? Won't they be 0, 1, and 2 naturally?